src/meshapi/views/forms.py

```diff
@@ -37,6 +37,7 @@ class JoinFormRequest:
     roof_access: bool
     referral: str
     ncl: bool
+    ContactPreference: str
```

Comment:
You will need to actually use this field in the logic below when constructing the member object, otherwise this will just get dropped on the floor